soundfile.py

```diff
@@ -307,12 +307,18 @@ def write(file, data, samplerate, subtype=None, endian=None, format=None,
 
         .. note:: The data type of *data* does **not** select the data
                   type of the written file. Audio data will be
-                  converted to the given *subtype*. Writing int values
-                  to a float file will *not* scale the values to
-                  [-1.0, 1.0). If you write the value ``np.array([42],
-                  dtype='int32')``, to a ``subtype='FLOAT'`` file, the
-                  file will then contain ``np.array([42.],
-                  dtype='float32')``.
+                  converted to the given *subtype*, with these caveats:
+
+                  * Writing int values to a float file will *not* scale the
+                    values to [-1.0, 1.0). If you write the value
+                    ``np.array([42], dtype='int32')``, to a ``subtype='FLOAT'``
+                    file, the file will then contain ``np.array([42.],
+                    dtype='float32')``.
+                  * For pure Python (non-Numpy) input values, floats will
```

Comment:
I think this wording could be misunderstood. I would rephrase it as "floats are expected in in the range ... and integers in the range ...".